test/stake.test.ts

```diff
@@ -51,8 +53,6 @@ describe('StakingPool.stake', () => {
   });
 
   context('when the pool initiated', async () => {
-    const stakeAmount = utils.parseEther('100');
-
     beforeEach('init the pool', async () => {
       await testEnv.rewardAsset.connect(deployer).faucet();
```

Comment:
The `faucet` and `approve` processes appear in every context. How about putting these snippets in the top hooks of its file to reduce duplicates?